tox.ini

```diff
@@ -87,12 +90,13 @@ commands =
 
 [testenv:check]
 deps =
-    docutils
     check-manifest
+    colorama
```

Comment:
Why do we need to add colorama ?

Reply:
https://github.com/pytest-dev/pytest-cov/actions/runs/4284044499/jobs/7460448953

Reply:
For future reference, when the GHA log is deleted:

```pytb
check: commands[3]> isort --check-only --diff src tests setup.py
Traceback (most recent call last):
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/main.py", line 87, in sort_imports
    incorrectly_sorted = not api.check_file(file_name, config=config, **kwargs)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/api.py", line [33](https://github.com/pytest-dev/pytest-cov/actions/runs/4284044499/jobs/7460448953#step:5:34)8, in check_file
    return check_stream(
           ^^^^^^^^^^^^^
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/api.py", line 272, in check_stream
    printer = create_terminal_printer(
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/format.py", line 153, in create_terminal_printer
    colorama.init(strip=False)
    ^^^^^^^^
NameError: name 'colorama' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/bin/isort", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/main.py", line 1225, in main
    for sort_attempt in attempt_iterator:
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/main.py", line 1209, in <genexpr>
    sort_imports(  # type: ignore
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/main.py", line 114, in sort_imports
    _print_hard_fail(config, offending_file=file_name)
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/main.py", line 127, in _print_hard_fail
    printer = create_terminal_printer(
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/pytest-cov/pytest-cov/.tox/check/lib/python3.11/site-packages/isort/format.py", line 1[53](https://github.com/pytest-dev/pytest-cov/actions/runs/4284044499/jobs/7460448953#step:5:54), in create_terminal_printer
    colorama.init(strip=False)
    ^^^^^^^^
NameError: name 'colorama' is not defined
check: exit 1 (0.09 seconds) /home/runner/work/pytest-cov/pytest-cov> isort --check-only --diff src tests setup.py pid=1722
  check: FAIL code 1 (7.13=setup[4.74]+cmd[0.[58](https://github.com/pytest-dev/pytest-cov/actions/runs/4284044499/jobs/7460448953#step:5:59),0.79,0.94,0.09] seconds)
  evaluation failed :( (7.19 seconds)
Error: Process completed with exit code 1.
```

Reply:
This looks like something that should be fixed upstream in isort rather than adding extra dependencies here.

We're installing isort 6.0.0b2 (released Dec 12, 2022) but looks like it was fixed in https://github.com/PyCQA/isort/pull/2032, merged Dec 12, 2022, which looks like it was only released in 5.11.1 (also Dec 12, 2022) - but there's no tags for the 6.x beta releases so it's hard to check.

Anyway, I asked in that PR if they can release a 6.x beta with the fix, then we can remove this extra dependency. 

Alternatively, we could test on the non-prerelease isort, but I don't know the original reasons for that, so maybe better to keep it.

Reply:
Comment added.